axelor-redmine/src/main/java/com/axelor/apps/redmine/imports/service/issues/RedmineImportTimeSpentServiceImpl.java

```diff
@@ -145,6 +163,8 @@ public void importTimeSpent(
 
         if (user != null) {
           this.userId = user.getId();
+          this.defaultCompanyId =
+              user.getActiveCompany() != null ? user.getActiveCompany().getId() : defaultCompanyId;
```

Comment:
Use new variable for active company as this logic update the defaultCompanyId with user.getActiveCompany().getId()
so when we use defaultCompanyId when user.getActiveCompany() is null then it gives wrong id